package.json

```
@@ -15,14 +15,17 @@
   },
   "devDependencies": {
     "@types/jest": "^24.0.15",
+    "browserify": "^16.5.1",
     "jest": "^24.8.0",
+    "mkdirp": "^1.0.4",
     "rimraf": "^2.6.3",
     "ts-jest": "^24.0.2",
+    "tsify": "^4.0.2",
     "typescript": "^3.5.2"
   },
   "scripts": {
     "prebuild": "rimraf dist",
-    "build": "tsc -p tsconfig.cjs.json & tsc -p tsconfig.esm.json & tsc -p tsconfig.dts.json",
+    "build": "tsc -p tsconfig.cjs.json & tsc -p tsconfig.esm.json & tsc -p tsconfig.dts.json & mkdirp dist/browser & browserify -e src/index-browser.ts -p [ tsify --noImplicitAny ] -o dist/browser/index.js",
```

```suggestion
    "build": "tsc -p tsconfig.cjs.json & tsc -p tsconfig.esm.json & tsc -p tsconfig.dts.json & browserify -e src/index-browser.ts -p [ tsify ] -o dist/browser/index.js",
```

As Browserify creates the out dir, the `mkdirp` can be skipped. Also `tsify` automatically reads the `tsconfig.json` file. The tsconfig at root has a whole lot of options, along with the `noImplicitAny` flag set. That can be skipped too.